war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -2344,26 +2386,6 @@ function validateButton(checkUrl,paramList,button) {
   });
 }
 
-function applyErrorMessage(elt, rsp) {
```

Comment:
Several plugins used this, including LDAP plugin: https://github.com/search?q=user%3Ajenkinsci+applyErrorMessage&type=Code